### ISSUE TITLE
Color Contrast of gray text in many screens fails the required color contrast ratio for Accessibility

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7632]
The gray text in many VPN screens fails the the required color contrast ratio (contrast of text against its background) for Accessibility, making it difficult for some users to read without switching to High Contrast themes. See attached screenshot:

!image-20230729-002316.png|width=1821,height=721!

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5294)
┆Reporter: Vinoo Cherian

**Comment by data-sync-user** [comment]
➤ Lesley Norton commented:

cc Peter Benvenuto

**Comment by data-sync-user** [comment]
➤ Peter Benvenuto commented:

We use gray-40 (#6D6D6E) for all our body text, and it passes all accessibility. If the text in those screens is not using gray-40 then it’s incorrect application of color. If you see the picture below, our body text color does pass necessary requirements:

!Screen Shot 2023-07-31 at 12.09.28 PM.png|width=1164,height=703!

CC Vinoo Cherian Lesley Norton Matt Lichtenstein

**Comment by data-sync-user** [comment]
➤ Matt Lichtenstein commented:

Vinoo Cherian I am having trouble understanding the screenshot. Is the tool flagging the app description text specifically? Because the version number just beneath it is the same color, but doesn’t appear to be flagged. They are both using gray-40 (#6D6D6E) as Peter Benvenuto mentioned.

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

Matt Lichtenstein The tool auto detects the contrast ratio of the text beneath the mouse cursor. In this instance, I captured the screen shot when it flagged the “A fast, secure and easy…” text. It also flags the version number and some of the other gray text in the app as failures. This happens on specific monitors, not all. I suspect that this is because the contrast ratio may be very close to the accepted 4:5:1. As for the accuracy of the reported color, I’m uncertain because it may be affected by the anti-aliasing that occurs during text rendering. However, it correctly detect the color in the ‘Suggestions color block’ in Peter’s screenshot as #6D6D6E, which also fails the contrast ratio in this tool.

Peter Benvenuto Accessibility Insights (https://accessibilityinsights.io/docs/windows/getstarted/colorcontrast/ ( https://accessibilityinsights.io/docs/windows/getstarted/colorcontrast/|smart-link )  shows that #6D6D6E against white background has contrast ratio of 4.03:1, so fails the contrast ratio test.

Maybe the contrast ratio is too close to the accepted standard, causing issues with certain tools, monitors, or font rendering.

**Comment by data-sync-user** [comment]
➤ Peter Benvenuto commented:

Vinoo Cherian The tool you are using is incorrect. The contrast ratio of our body text is 5.16:1, I did a test in the browser by changing the font color on our landing page to #6D6D6E if you look at “A virtual private network from the makers of Firefox.”. Then look at the accessibility contrast in the accessibility tab in Firefox, where it correctly says the ration is 5.17:1 with a passing check

This is a non-issue imo, and may be an issue with the tool you are using.

Matt Lichtenstein Lesley Norton

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

Peter Benvenuto, yes, looks like tool seems to be having some problems. I’ll cancel this issue. Thanks.

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

This color passes in other tools. It also passes in the Accessibility Insights tool sometimes.

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled
